drivers/src/soc_ifc.rs

```diff
@@ -131,6 +132,85 @@ impl SocIfc {
         let flags: MfgFlags = soc_ifc_regs.cptra_dbg_manuf_service_reg().read().into();
         flags.contains(MfgFlags::GENERATE_IDEVID_CSR)
     }
+
+    /// Enbale or disable WDT1
+    ///
+    /// # Arguments
+    /// * `enable` - Enable or disable WDT1
+    ///
+    pub fn configure_wdt1(&mut self, enable: bool) {
+        let soc_ifc_regs = self.soc_ifc.regs_mut();
+        soc_ifc_regs
+            .cptra_wdt_timer1_en()
+            .write(|w| w.timer1_en(enable));
+    }
+
+    fn get_cycle_count(&self, seconds: u32) -> CaliptraResult<u64> {
```

Comment:
Why do we care about seconds vs cycles? AFAICT caliptra takes N cycles to accomplish a particular task regardless of clock rate.

Reply:
Removed cycle count computation.